context/test_notebooks.py

```diff
@@ -30,7 +30,9 @@
     'cuspatial/cuproj_benchmark.ipynb',
     # context on these being skipped: https://github.com/rapidsai/cuspatial/pull/1407
     'cuspatial/cuspatial_api_examples.ipynb',
-    'cuspatial/nyc_taxi_years_correlation.ipynb'
+    'cuspatial/nyc_taxi_years_correlation.ipynb',
+    # context on skip zipcodes: https://github.com/rapidsai/cuspatial/issues/1426
+    'cuspatial/ZipCodes_Stops_PiP_cuSpatial.ipynb',
 ]
```

Comment:
I support this, but unless there's been a fix in `cuml` that I'm not aware of, I don't think it'll be enough to unblock CI here.

There's a `cuml` notebook failing in the most recent `publish` job for `branch-24.08` as well:

> Completed cuml/forest_inference_demo.ipynb with 1 warnings and 6 errors

([build link](https://github.com/rapidsai/docker/actions/runs/10244736365/job/28356773321))

Unless we're planning to fix that on 24.08 for `cuml`, that `cuml/forest_inference_demo.ipynb` should be skipped as well.

Reply:
Do you have a link to a cuML issue describe this notebook failure? If not, could you please file one and link it here?

Reply:
I don't, just noticed it in the logs you shared. Will put up an issue link shortly.

Reply:
Put up https://github.com/rapidsai/cuml/issues/6008.

Reply:
```suggestion
    # context on skip forest inference: https://github.com/rapidsai/cuml/issues/6008
    'cuml/forest_inference_demo.ipynb',
]
```

Reply:
Thanks James! 🙏

Have update the PR accordingly. Please take another look